HW03/Lizaveta_Muromskaya/lizaveta_muromskaya_hw_03_t_03.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+TIME = /[0-9]+:[0-9]+:[0-9]+\.[0-9]/.freeze
+def time_in_sec(time, seconds)
+  time.each do |time_numbers|
+    time_buf = time_numbers.split(':')
+    sec = time_buf[0].to_f * 3600 + time_buf[1].to_f * 60 + time_buf[2].to_f
+    seconds << sec
+  end
+end
```

Comment:
This method also shouldn't modify `seconds` 😉 
```suggestion
def time_in_sec(time)
  time.map do |time_numbers|
    time_buf = time_numbers.split(':')
    time_buf[0].to_f * 3600 + time_buf[1].to_f * 60 + time_buf[2].to_f
  end
end

seconds = time_in_sec(time)
```